lib/bitgo_client/v2.rb

```diff
@@ -25,59 +25,59 @@ def client
       BitgoClient::Client.new(access_token)
     end
 
-    def create_address(wallet_id, coin_code: :tbtc, logger: nil)
-      client.request("#{base_path}/#{coin_code}/wallet/#{wallet_id}/address", method: :post, logger: logger)
+    def create_address(wallet_id, coin_code: :tbtc, logger: nil, proxy: nil)
```

Comment:
I think it'll look much cleaner if the proxy is passed when initializing the client, so that you don't need to pass it to each of these client methods.

I don't see a case where we want to make some calls behind a proxy and others not. Sounds like setting the proxy on the client instance is a safe bet to take.

Reply:
Ah yes makes sense!